.github/workflows/check-apps.yml

```diff
@@ -116,6 +108,7 @@ jobs:
         run: |
           rustup default ${{ env.RUST_VERSION }}
           rustup target install wasm32-unknown-unknown
+          cargo install cargo-concordium@${{ env.CARGO_CONCORDIUM_VERSION }}
```

Comment:
Installing cargo-concordium can take a while, which is why we download the binary in several places.
Maybe less worth it due to caching, but it can be [downloaded from here now](https://github.com/Concordium/concordium-smart-contract-tools/releases/tag/releases%2Fcargo-concordium%2F4.2.0)